docs/UserGuide.md

```diff
@@ -71,7 +112,43 @@ open the help window.<br>
 
 ## Navigating the User Interface
 
-placeholder
+![NavigatingUi](images/NavigatingUserInterface.png)
+The user interface is split into the input and output sections, as well as
+two view columns for **Person List** and **Overview**.
+
+Item | Description
+--------|------------------
+**command box** | This is where you enter your commands to DengueHotspotTracker.
+**output box** | The output box displays additional information from a command. <br> :bulb: For a *successful* command, the output box may contain information such as the number of entries listed. <br> :bulb: For an *unsuccessful* command, such as invalid input, the output box will give detailed feedback about which part of the command is invalid and how to fix it.
+**person list** | A list of people and their details.
+**overview** | Provides an overview of the total number of cases and cases per category.
+
+<div style="page-break-after: always;"></div>
+Here is the breakdown of an individual *person card*.
+
+![PersonCardUi](images/PersonCardUi.png)
+
+Item | Description
+--------|------------------
+**id** | Current Index of the person in the list.
+**Name** | Name of person who got dengue.
+**Age** | Age of the person who got dengue.
+**Postal** | Postal code of the person who got dengue.
+**Date** | Date that the person got dengue.
+**Variant** | Dengue Variant person got (DENV1, DENV2 etc.).
+
+<div style="page-break-after: always;"></div>
+Here is the breakdown of an individual *bin card*.
+
+![OverviewCardUi](images/OverviewCardUi.png)
+
+Item | Description
+--------|------------------
+**id** | Index in the overview list.
+**Overview Category** | Overview category being currently displayed.  <br> :bulb: The default overview category is by location.
```

Comment:
```suggestion
**Overview Category** | Overview category currently being displayed.  <br> :bulb: The default overview category is by location.
```

Reply:
Also, in the person card description above, all the items were written in lowercase in both the image and the table! It might be better to standardise case for both person card and bin card (for both the images and the tables)—maybe everything in sentence case?